Integrate-Kobiton.md

```diff
@@ -0,0 +1,257 @@
+# Run Kobiton Automation Test on CircleCI
+## Table of contents
+- [Prerequisites](#prerequisites)
+
+- [1. Configuring GitHub repository to be used with CircleCI](#1-configuring-github-repository-to-be-used-with-circleci)
+
+- [2. Getting required parameters](#2-getting-required-parameters)
+
+- [3. Configuring automation test script](#3-configuring-automation-test-script)
+
+- [4. Configuring CircleCI with Kobiton](#4-configuring-circleci-with-kobiton)
+
+- [5. Executing the automation test](#5-executing-the-automation-test)
+
+- [6. Fetch test session data using REST API](#6-fetch-test-session-data-using-rest-api)
+
+## Prerequisites
+- A Kobiton account.
+    > If your don't have one, go to [https://portal.kobiton.com/register](https://portal.kobiton.com/register) to create an account.
+- A GitHub account that will be used to login to CircleCI
+    > If you don't have one, go to [https://github.com/join](https://github.com/join) to create an account.
+- A GitHub repository to be used for storing automation testing script(s). CircleCI will automatically execute that script(s), which will execute the test on Kobiton devices.
+
+## 1. Configuring GitHub repository to be used with CircleCI
```

Comment:
What user needs is to **have is an existing repo integrating with `CircleCI`**, we provide **no training for how to use `CircleCI` here**. There is an assumption that CircleCI is required, and now we give a bunch of text educating user how to use `CircleCI` !